src/platform-includes/getting-started-primer/flutter.mdx

```diff
@@ -11,6 +11,8 @@ Features:
   - ObjC, Swift, and C for iOS and macOS
   - Flutter and Android encapsulated exceptions (PlatformException)
 - Offline storage of events
+  - Android: Offline caching when a device is offline; we send a report once the application is restarted
+  - iOS: Offline caching when a device is unable to connect; we send a report once we receive another event 
```

Comment:
Maybe worth that to RN, Capacitor, and on Android/iOS as well.

Reply:
Good idea, I'll add those to this PR

Reply:
- Added clarification for handling of mobile offline storage to React Native and Cordova ✔️ 

It looks like Capacitor [has solid coverage](https://docs.sentry.io/platforms/javascript/guides/capacitor/configuration/integrations/plugin/#offline) for how it handles offline storage, so do [iOS](https://docs.sentry.io/platforms/apple/guides/ios/) and [Android](https://docs.sentry.io/platforms/android/)